cfgov/jinja2/v1/_includes/molecules/related-content.html

```diff
@@ -42,10 +42,11 @@ <h2 class="a-heading">
         {% for link in value.links %}
             <li class="m-list_item">
                 <a href="{{ link.url }}"
-                   class="m-list_link"
-                   {% if link.aria_label %}aria-label="{{ link.aria_label }}"{% endif %}>
-                    {{ link.text }}
-                </a>
+                  class="m-list_link"
+                  {%- if link.aria_label -%}
+                    aria-label="{{ link.aria_label }}"
+                  {% endif -%}
+                >{{ link.text }}</a>
```

Comment:
Not super keen on the floating `>` but I guess that's just as weird as putting it on the prior line
```
{% if link.aria_label %}aria-label="{{ link.aria_label }}"{% endif %}>
   {{ link.text }}</a>
```

Reply:
Prior line or allowing it to have its own line would be more readable for me. If I ever looked back on this code, I would probably miss `>` in its current position.